api/src/main/java/com/github/streamshub/console/api/security/ConsoleAuthenticationMechanism.java

```diff
@@ -102,6 +102,10 @@ boolean oidcEnabled() {
 
     @Override
     public Uni<SecurityIdentity> authenticate(RoutingContext context, IdentityProviderManager identityProviderManager) {
+        if (!context.normalizedPath().startsWith("/api")) {
+            return Uni.createFrom().nullItem();
+        }
```

Comment:
No authentication for non-API paths. E.g. `/metrics`, `/health`, and `/openapi`

Reply:
I think security best practice would be to flip it the other way around.   i.e list of the paths that allowed without auth and let everything else require it.

Reply:
Done - replaced the `!context.normalizedPath().startsWith("/api")` with a check against a predefined set of paths that should not be authenticated.